loadimpact/loadimpact.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+// An example of usign Pantheon's Quicksilver technology to do 
+// a performance test using Load Impact
+
+// Provide the API Key provided by Load Impact
+// For extra security, you can store this information in
+// the private area of the files directory as documented
+// at https://github.com/pantheon-systems/quicksilver-examples.
+$api_key = 'add-api-key-here';
```

Comment:
The advantage of using secrets.json is that you can then install this example via `terminus quicksilver install`, and set your secrets via `terminus secrets set key value`, with no need to alter the code.

The code as written has the advantage of being short, and not requiring any duplicated functions to manage secrets. So, right now there is a fair bit of duplicate code in the Quicksilver documentation regarding secrets. There is a discussion on that topic in #74.